DEBUG E27 - wypisz kazdy komunikat wifili host->fw (typ, interfejs, wynik)
do dmesg. Ma pokazac dokladna sekwencje przy asocjacji klienta i OSTATNI
komunikat przed zwisem phys_if. Nie wysylac.

--- a/nss_wifili.c
+++ b/nss_wifili.c
@@ -235,7 +235,12 @@
 		return NSS_TX_FAILURE;
 	}

-	return nss_core_send_cmd(nss_ctx, msg, sizeof(*msg), NSS_NBUF_PAYLOAD_SIZE);
+	{
+		nss_tx_status_t st = nss_core_send_cmd(nss_ctx, msg, sizeof(*msg), NSS_NBUF_PAYLOAD_SIZE);
+		pr_info("wifili-tx: type=%u if=%u len=%u st=%d\n",
+			ncm->type, ncm->interface, ncm->len, st);
+		return st;
+	}
 }
 EXPORT_SYMBOL(nss_wifili_tx_msg);

